js/ui/form/ui.form.layout_manager.utils.js

```diff
@@ -8,6 +8,7 @@ import Guid from '../../core/guid';
 import { SIMPLE_ITEM_TYPE } from './constants';
 
 const EDITORS_WITH_ARRAY_VALUE = ['dxTagBox', 'dxRangeSlider'];
+export const EDITORS_WITHOUT_LABELS = ['dxCalendar', 'dxCheckBox', 'dxHtmlEditor', 'dxRadioGroup', 'dxRangeSlider', 'dxSlider', 'dxSwitch'];
```

Comment:
The opposite approach is EDITORS_**WITH**_LABELS
Why the used is better?

Reply:
Editors without labels are the exception case for form rendering.
So I think it's better to specify exceptions items explicitly.

If we add a new editor, then with a high degree of probability it will be a common editor,
and it will not need to be added to this list.

Also, count of editor without labels is less then with labels.
So, I decided to specify  editors without labels